.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,5 +15,10 @@ Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
 - [ ] documentation is changed or added
 - [ ] commit message follows commit guidelines
 
+<!-- If this is your first contribution to Node.js, please also check the
```

Comment:
That would work also. Making them checkboxes makes it more likely that someone will actually pay attention to the additional items tho.